README.md

```diff
@@ -66,6 +66,9 @@ fill them in
         $ python api/cfc_webapp.py
 
    You may need to install some of the python dependencies from above
+   Amongst the dependencies include:
+   -cherrypy
```

Comment:
I believe you shouldn't need cherrypy unless you are running over SSL. Did you have an issue with cherrypy when you started with the default config?